xdsl/interactive/app.py

```diff
@@ -51,6 +55,55 @@
 ALL_PASSES = tuple(sorted((p_name, p()) for (p_name, p) in get_all_passes().items()))
 """Contains the list of xDSL passes."""
 
+ALL_PATTERNS = tuple(
+    (op_name, pattern_name, pattern)
+    for (op_name, pattern_by_name) in individual_rewrite.REWRITE_BY_NAMES.items()
+    for (pattern_name, pattern) in pattern_by_name.items()
+)
+"""Contains all the rewrite patterns."""
+
+NamedTuple = tuple[str, str]
```

Comment:
Sorry, by NamedTuple I meant the Python NamedTuple: https://docs.python.org/3/library/collections.html#collections.namedtuple
As it is, the name is a little confusing due to the clash